tests/go/GofmtBearTest.py

```diff
@@ -5,3 +5,11 @@
     GofmtBear,
     ('package main\n\nfunc main() {\n\treturn 1\n}',),
     ('package main\nfunc main() {\n\treturn 1\n}',))
+
+GofmtBearWithSettings = verify_local_bear(
+    GofmtBear,
+    ('package main\n\nfunc main() {\n\tarray := []int{0, 2}' +
+        '\n\tarray2 := array[1:]\n\treturn 1\n}',),
+    ('package main\n\nfunc main() {\n\tarray := []int{0, 2}' +
+        '\n\tarray2 := array[1:len(array)]\n\treturn 1\n}',),
+    settings={'simplify': 'true'})
```

Comment:
I had already tried it before and it didn't work out. 
Also other tests (like GNUIndent) are using 'true' as string so this is consistent with them.